### PR TITLE
Make contacts between all activities

### PR DESCRIPTION
**Pull request by tuval190**
- Make contacts between all activities.
- Set dimens.
- Unite the design in all the activities.

### CODE REVIEW

File: app/src/main/java/com/example/secondproject/FriendProfileActivity.java [hunk 1]
```diff
@@ -11,4 +13,14 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_friend_profile);
     }
+
+    public void launchFriendsPageActivity(View view) {
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendProfileActivity.java:17]: Good name right as we said

File: app/src/main/java/com/example/secondproject/FriendProfileActivity.java [hunk 2]
```diff
@@ -11,4 +13,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        Intent intent = new Intent(this, FriendsPageActivity.class);
+        startActivity(intent);
+    }
+
+    public void launchFriendShazamzamsActivity(View view) {
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendProfileActivity.java]: same as above

File: app/src/main/java/com/example/secondproject/FriendProfileActivity.java [hunk 3]
```diff
@@ -11,4 +13,14 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_friend_profile);
     }
+
+    public void launchFriendsPageActivity(View view) {
+        Intent intent = new Intent(this, FriendsPageActivity.class);
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendProfileActivity.java:18]: Please change the name of the intent to something more indicative and as we agreed about the variables names. something like friends_page_intent will be better.

File: app/src/main/java/com/example/secondproject/FriendProfileActivity.java [hunk 4]
```diff
@@ -11,4 +13,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        startActivity(intent);
+    }
+
+    public void launchFriendShazamzamsActivity(View view) {
+        Intent intent = new Intent(this, FriendShazamzamsActivity.class);
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendProfileActivity.java:23]: Same as above

File: app/src/main/java/com/example/secondproject/FriendShazamzamsActivity.java [hunk 5]
```diff
@@ -11,4 +13,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_friend_shazamzams);
     }
+
+    public void launchFriendProfileActivity(View view) {
+        Intent intent = new Intent(this, FriendProfileActivity.class);
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendShazamzamsActivity.java:18]: Same as above

File: app/src/main/java/com/example/secondproject/FriendsPageActivity.java [hunk 6]
```diff
@@ -47,4 +47,9 @@ public void launchMyProfilePage(View view) {
         Intent myProfileIntent = new Intent(this, MyProfileActivity.class);
         startActivity(myProfileIntent);
     }
+
+    public void launchFriendProfileActivity(View view) {
+        Intent myProfileIntent = new Intent(this, FriendProfileActivity.class);
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/FriendsPageActivity.java:52]: Looking good, better than those above, but named it as we agreed about vars name, for example "my_profile_intent".

File: app/src/main/java/com/example/secondproject/HomePageActivity.java [hunk 7]
```diff
@@ -35,4 +37,9 @@ public void hum_1_clicked(View view) {
     public void hum_2_clicked(View view) {
         answer2EditText.setVisibility(View.VISIBLE);
     }
+
+    public void launchMyProfilePage(View view) {
+        Intent intent = new Intent(this, MyProfileActivity.class);
```
Comment (baraksw) [inline app/src/main/java/com/example/secondproject/HomePageActivity.java:42]: same as above about the intents names.

File: app/src/main/res/layout/activity_friend_profile.xml [hunk 8]
```diff
@@ -65,26 +77,28 @@
...
-
     <TextView
-        android:id="@+id/shazamzams_textView"
-        android:layout_width="@dimen/shazamzams_textView_width"
-        android:layout_height="@dimen/shazamzams_textView_height"
-        android:text="@string/shazamzams_text"
-        android:textAppearance="@style/TextStyle"
+        android:id="@+id/xp_points_textView5"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
```
Comment (baraksw) [inline app/src/main/res/layout/activity_friend_profile.xml:83]: Why did you change the sizes of the TextView?

File: app/src/main/res/layout/activity_friend_profile.xml [hunk 9]
```diff
@@ -65,26 +77,28 @@
...
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        android:layout_marginBottom="@dimen/shazamzams_textView_bottom_margin"/>
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <ImageButton
+        android:id="@+id/speaker_imageView"
+        android:layout_width="64dp"
+        android:layout_height="58dp"
```
Comment (baraksw) [inline app/src/main/res/layout/activity_friend_profile.xml:94]: Add them to dimense and I think they should be equal because it's a square.

File: app/src/main/res/layout/activity_home_page.xml [hunk 10]
```diff
@@ -8,55 +8,54 @@
...
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/mic_imageButton"
-        app:srcCompat="@drawable/piano_image" />
+        android:gravity="center"
+        android:text="@string/my_profile_text"
+        app:layout_constraintEnd_toEndOf="@+id/profile_link_imageButton"
+        app:layout_constraintHorizontal_bias="0.592"
+        app:layout_constraintStart_toStartOf="@+id/profile_link_imageButton"
+        app:layout_constraintTop_toBottomOf="@+id/profile_link_imageButton" />
```
Comment (baraksw) [inline app/src/main/res/layout/activity_home_page.xml:20]: It's should be without the '+' after the '@' symbol

File: app/src/main/res/layout/activity_home_page.xml [hunk 11]
```diff
@@ -8,55 +8,54 @@
...
 
     <ImageButton
         android:id="@+id/mic_imageButton"
         android:layout_width="260dp"
         android:layout_height="260dp"
+        android:background="@android:color/transparent"
         android:scaleType="centerCrop"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.496"
```
Comment (baraksw) [inline app/src/main/res/layout/activity_home_page.xml:29]: Add it to dimens.

File: app/src/main/res/layout/activity_login.xml [hunk 12]
```diff
@@ -78,19 +79,6 @@
         app:srcCompat="@drawable/lets_go_button"
         android:layoutDirection="ltr" />
 
-    <ImageButton
-        android:id="@+id/loginBtn2"
-        android:layout_width="110dp"
-        android:layout_height="55dp"
-        android:layout_marginBottom="250dp"
```
Comment (baraksw) [inline app/src/main/res/layout/activity_login.xml:85]: Add to dimens.

File: app/src/main/res/layout/activity_my_profile.xml [hunk 13]
```diff
@@ -8,33 +8,33 @@
...
-        android:onClick="launchHomePage"/>
+        app:layout_constraintTop_toTopOf="parent" />
 
     <ImageButton
         android:id="@+id/my_friend_button"
-        android:layout_width="@dimen/my_friends_button_width"
-        android:layout_height="@dimen/my_friends_button_height"
-        android:layout_marginTop="@dimen/global_top_margin_type2"
+        android:layout_width="60dp"
+        android:layout_height="36dp"
```
Comment (baraksw) [inline app/src/main/res/layout/activity_my_profile.xml:29]: why you removed the resource dimens instead of changing their values?

**Review by baraksw — APPROVED** [review]

**Comment by tuval190** [comment]
Thanks for the review I'll fix all the comments.
